verilog/analysis/checkers/endif_comment_rule_test.cc

```diff
@@ -106,6 +107,19 @@ TEST(EndifCommentRuleTest, RejectsEndifWithoutComment) {
   RunLintTestCases<VerilogAnalyzer, EndifCommentRule>(kTestCases);
 }
 
+TEST(EndifCommentRuleTest, ApplyAutoFix) {
+  // Alternatives the auto fix offers
+  constexpr int kEOLComment = 0;
+  constexpr int kBlocComment = 1;
```

Comment:
kBloc**k**Comment ?

Reply:
Done, called it `kBlkComment` so that it fits in the line below.